#! /usr/bin/env python

PACKAGE='wayspid'

import roslib;
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       	Name			Type      	Level	Description				Default 	Min	Max
gen.add(	"record_start",		bool_t,		0,	"Data record start flag",		False)
gen.add( 	"max_rec_time",		int_t, 		0, 	"Max time to rec data (s)", 		35, 		10, 	100)

exit(gen.generate(PACKAGE, "wayspid", "setDataRecord"))
